.palantir/revapi.yml

```diff
@@ -297,10 +297,6 @@ acceptedBreaks:
         \ T) throws java.io.IOException, com.fasterxml.jackson.core.JacksonException\
         \ @ org.apache.iceberg.rest.RESTSerializers.UpdateRequirementDeserializer"
       justification: "False positive - JacksonException is a subclass of IOException"
-    - code: "java.method.removed"
-      old: "method <T> org.apache.iceberg.io.CloseableIterable<T> org.apache.iceberg.deletes.Deletes::filterDeleted(org.apache.iceberg.io.CloseableIterable<T>,\
-        \ java.util.function.Predicate<T>)"
-      justification: "Removing deprecations for 1.2.0"
```

Comment:
Not sure why this changed....taking a look

Reply:
yeah that shouldn't change. just FYI that I've basically removed all deprecations across iceberg-core in https://github.com/apache/iceberg/pull/7156 and I'm not seeing that change to revapi.yml. I'll wait for you to merge your PR and rebase mine then

Reply:
fixed!